routes/classes.js

```diff
@@ -0,0 +1,63 @@
+const express = require("express")
+const cors = require("cors")
+const School = require("../School")
+const Student = require("../Student")
+let classes = express.Router()
+classes.use(cors())
+let mySchool = new School();
+classes.get("/", (request, response) => {
+    response.json(mySchool)
+})
+
+classes.post("/", (request, response) => {
+    let classes = request.body.classes;
+    let teacher = request.body.teacher;
+    if(classes.length===0|| teacher.length===0){
+        response.json({message: "Please fill out all the information or Class already exists"})
+    }else{
+    mySchool.addClass(classes, teacher,)
+    response.json({
+        "classes": { "nameOfClass": classes, "teacher": teacher ,"students": []},
+        "message": `Created a ${classes} class taught by ${teacher}`,
```

Comment:
Here you are returning your request inside of your response. That is not technically correct. You might create an illusion that everything went through great even if it didn't. You need to return in your addClass function and then assign the return value of that function to a variable here on line 18. 
Then you include it in the response object.